### PR TITLE
MF4 Log Parsing

### PR DESCRIPTION
**Pull request by jpatch22**
### Summary
Add parsing for mf4 files to the backend - log files go in the added folder and the output is saved as a csv
### Checklist
*Please change `[ ]` to `[x]` when you are ready.*
- [ ] I have read and followed the code conventions detailed in [README.md](../README.md) (*This will save time for both you and the reviewer!*).
- [ ] If this pull request is longer then **500** lines, I have provided *explicit* justification in the summary above explaining why I *cannot* break this up into multiple pull requests (*Small PR's are faster and less painful for everyone involved!*).

### CODE REVIEW

File: software/tracksight/backend/app/mf4_logs/README.md
Comment (forbesashli) [inline software/tracksight/backend/app/mf4_logs/README.md:1]: maybe put in a small description and then rebase (?)

**Review by forbesashli — COMMENTED** [review]
overall lgtm

**Review by forbesashli — APPROVED** [review]
nvm that was a nit pick

File: software/tracksight/backend/app/mf4_logs/README.md
Comment (jpatch22) [reply software/tracksight/backend/app/mf4_logs/README.md:1]: sg

**Pull request merged**